src/StructuredLogViewer/Controls/BuildControl.xaml.cs

```diff
@@ -386,7 +386,7 @@ private void PopulateTimeline()
         {
             if (this.timeline.Timeline == null)
             {
-                var timeline = new Timeline(Build);
+                var timeline = new Timeline(Build, false);

```

Comment:
let's add the named argument: `includeCpp: false`

In general I prefer to include named arguments when passing literals such as `true`, `false` and `null`.